src/components/swap/SwapPlatformSelector.tsx

```diff
@@ -86,95 +112,109 @@ export function SwapPlatformSelector({
-                        : `${slippageAdjustedAmounts[Field.INPUT]?.toSignificant(4)}` ?? '-'}
-                    </TYPE.subHeader>
-                    <CurrencyLogo
-                      currency={isExactIn ? trade.outputAmount.currency : trade.inputAmount.currency}
-                      size="14px"
-                      marginLeft={4}
-                    />
-                  </RowFixed>
-                </td>
-              </tr>
-            )
-          })}
-        </tbody>
-      </Table>
+    <AutoColumn>
       {selectedTrade instanceof UniswapV2Trade && selectedTrade.route.path.length > 2 && (
-        <Flex mx="2px" width="100%">
-          <Flex>
-            <Box>
-              <TYPE.body fontSize="12px" lineHeight="15px" fontWeight="500" minWidth="auto">
-                Route
-              </TYPE.body>
-            </Box>
-            <Box>
-              <QuestionHelper text="Routing through these tokens resulted in the best price for your trade." />
-            </Box>
-          </Flex>
+        <StyledRouteFlex>
+          {!isMobileByMedia && (
+            <TYPE.body fontSize="14px" lineHeight="15px" fontWeight="400" minWidth="auto" color={theme.purple2}>
+              Route
+            </TYPE.body>
+          )}
           <Box flex="1">
             <SwapRoute trade={selectedTrade} />
           </Box>
+        </StyledRouteFlex>
+      )}
+      <SelectionListWindowWrapper>
+        <SelectionListLabelWrapper>
+          <SelectionListLabel justify={true} flex={showGasFees ? '30%' : '45%'}>
+            {t('exchange')}
+          </SelectionListLabel>
+          <SelectionListLabel>{isMobileByMedia ? t('pImp') : t('pImpact')}</SelectionListLabel>
+          <SelectionListLabel>{t('fee')}</SelectionListLabel>
+          {showGasFees && <SelectionListLabel>{t('gas')}</SelectionListLabel>}
+          <SelectionListLabel flex="25%">
+            {independentField === Field.OUTPUT ? t('maxSent') : t('minReceived')}
+          </SelectionListLabel>
+        </SelectionListLabelWrapper>
+
+        {displayedPlatformTrade?.map((trade, i) => {
+          if (!trade) return null // some platforms might not be compatible with the currently selected network
+          const isExactIn = trade.tradeType === TradeType.EXACT_INPUT
+          const gasFeeUSD = gasFeesUSD[i]
+          const { realizedLPFee, priceImpactWithoutFee } = computeTradePriceBreakdown(trade as UniswapV2Trade)
+          const slippageAdjustedAmounts = computeSlippageAdjustedAmounts(trade)
+          const tokenAmount = limitNumberDecimalPlaces(
+            isExactIn ? slippageAdjustedAmounts[Field.OUTPUT] : slippageAdjustedAmounts[Field.INPUT]
+          )
+
+          return (
+            <SelectionListOption
+              key={trade.platform.name}
+              isSelected={selectedTrade?.platform.name === trade.platform.name}
+              isLoading={false}
```

Comment:
Where else is this property computed? Perhaps we should remove it.

Reply:
This is being used in EcoBridge when waiting for option results :)